projects/wave-core/src/lib/operators/types/gdal-source-type.model.ts

```diff
@@ -62,12 +62,17 @@ export class GdalSourceType extends OperatorType {
                     return {
                         channel: channelDict.channel,
                         datatype: channelDict.datatype,
-                        file_name: channelDict.file_name,
+                        file_name: channelDict.file_name ? channelDict.file_name : undefined,
+                        path: channelDict.path ? channelDict.path : undefined,
                         unit: Unit.fromDict(channelDict.unit),
+                        netcdf_subdataset: channelDict.netcdf_subdataset ? channelDict.netcdf_subdataset : undefined,
```

Comment:
Why is it necessary to have
*if not undefined, add the variable, else undefined*?
What was the problem with the setting before?

Reply:
variable is not required by mapping here

Reply:
Yes, but if `channelDict.path` is `undefined`, then `path` is already `undefined` if you say `{…, path: channelDict.path, …}`.

Reply:
ok